### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by I-Have-No-Idea-What-IAmDoing**
I changed some out-dated/inaccurate information

### CODE REVIEW

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
I also messed up with the commits, so it best to squash this as the first 3 really are just getting my fork up to speed

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
also added html color codes support

File: README.md [hunk 1]
```diff
@@ -117,25 +121,24 @@ func _on_SyndiBox_signal_tag(identifier):
...
-    string = string.insert(sb.step,char(8203) + "[:2][^r]")
+  "[Magenta]": # Example
+    string.erase(sb.step,sb.emph.length())
+    string = string.insert(sb.step,char(8203))
     sb.color = Color(255, 0, 255, 1)
 ```  
-This very simple example turns the text magenta when you use the tag `[Xm]` in a Dialog string.  
-As custom tags are checked and processed last, you `X` tags will override any other tags if they conflict. 
+This very simple example turns the text magenta when you use the tag `[Magenta]` in a Dialog string.  
+As custom tags are checked and processed first, your custom tags will be overriden any other tags if they conflict.
```
Comment (Sudospective) [inline README.md]: Will these custom tags _override_ or _be overriden_? It seems like a very big difference in wording and I'd like some clarification before I approve.

**Review by Sudospective — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zaknafean) [reply README.md]: I didn't write the code, but looking at the code, custom code is definitely overwritten as it processes first. I mistakenly wrote 'overrides' in the previous version of the readme based on the comments of the code which were out dated, not the code itself. Trust the guy who wrote it (him) not the guy who just tried to document it (me). Relevant code below.

```gdscript
func emph_check(string): # Called before printing each step
	# Checks if if the start of a tag
	if string[step] == "[":
		# Search for the entire tag
		var tag_match = tag_checker.search(string,step)
		# If it is found then saves it to emph
		if tag_match != null:
			emph = tag_match.get_string()
			# Attempt a match for a valid tag in our current string.
			if CUSTOM_EFFECTS:
				string = custom.check(string)
			string = speaker_check(string)
			string = font_check(string)
			string = color_check(string)
			string = speed_check(string)
			string = pos_check(string)
			string = pause_check(string)
			string = hide_check(string)
			string = signal_check(string)
```

Comment (Sudospective) [reply README.md]: Yeah, I had custom tags set first so people could use it as a base for custom presets (i.e., characters or custom/multiple effects), and they are checked first, and overridden by any tags following it before the reset tag. So, if I were to write `[X:][`c]`, any custom color I set would be overridden by the built-in tag for red.

**Review by Sudospective — APPROVED** [review]

**Pull request merged**